centrifuge-app/src/pages/Pool/Liquidity/index.tsx

```diff
@@ -77,22 +152,44 @@ export const PoolDetailLiquidity: React.FC = () => {
       </PageSection>
       <PageSection
         title={`Epoch ${pool.epoch.current}`}
-        titleAddition={pool.epoch.isInSubmissionPeriod ? 'Calculating orders...' : 'Ongoing'}
+        titleAddition={
+          isInSubmissionPeriod || isInExecutionPeriod || isInChallengePeriod ? 'Calculating orders...' : 'Ongoing'
+        }
         headerRight={
           <Shelf gap="1">
-            {!pool.epoch.isInSubmissionPeriod && (
+            {!isInSubmissionPeriod && !isInChallengePeriod && !isInExecutionPeriod && (
               <Tooltips type="epochTimeRemaining" label={`${hours} hrs and ${minutes} min remaining`} />
             )}
-            <Button
-              small
-              variant="secondary"
-              onClick={closeEpoch}
-              disabled={!pool}
-              loading={pool.epoch.isInSubmissionPeriod}
-              loadingMessage="Calculating..."
-            >
-              Close
-            </Button>
+            {isInChallengePeriod && (
+              <Text variant="label2">
+                Epoch is in challenge period until block {challengePeriodEnd.toString()} is finalized
```

Comment:
1) I think we want to use the term calculation period rather than challenge period (but maybe check with Esther/Nico on this)
2) I think we should convert this to something like
```suggestion
                Epoch is in challenge period until {(challengePeriodEnd - currentBlock) * avgTimePerBlock / 60} min from now
```

Reply:
That's a good suggestion. I spoke with Esther about it and we've decided to remove it for now and tackle the count down in a separate ticket